File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/openTripMap/Bbox.java
package com.tammamkhalaf.myuaeguide.categories.openTripMap;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Bbox {

    @SerializedName("lat_max")
    @Expose
    private Double latMax;
    @SerializedName("lat_min")
    @Expose
    private Double latMin;
    @SerializedName("lon_max")
    @Expose
    private Double lonMax;
    @SerializedName("lon_min")
    @Expose
    private Double lonMin;

    public Double getLatMax() {
        return latMax;
    }

    public void setLatMax(Double latMax) {
        this.latMax = latMax;
    }

    public Double getLatMin() {
        return latMin;
    }

    public void setLatMin(Double latMin) {
        this.latMin = latMin;
    }

    public Double getLonMax() {
        return lonMax;
    }

    public void setLonMax(Double lonMax) {
        this.lonMax = lonMax;
    }

    public Double getLonMin() {
        return lonMin;
    }

    public void setLonMin(Double lonMin) {
        this.lonMin = lonMin;
    }

}

File: kmmsharedmodule/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/openTripMap/Bbox.kt
package com.tammamkhalaf.myuaeguide.categories.openTripMap

import com.google.gson.annotations.Expose

class Bbox {
    @SerializedName("lat_max")
    @Expose
    private var latMax: Double? = null

    @SerializedName("lat_min")
    @Expose
    private var latMin: Double? = null

    @SerializedName("lon_max")
    @Expose
    private var lonMax: Double? = null

    @SerializedName("lon_min")
    @Expose
    private var lonMin: Double? = null
    fun getLatMax(): Double? {
        return latMax
    }

    fun setLatMax(latMax: Double?) {
        this.latMax = latMax
    }

    fun getLatMin(): Double? {
        return latMin
    }

    fun setLatMin(latMin: Double?) {
        this.latMin = latMin
    }

    fun getLonMax(): Double? {
        return lonMax
    }

    fun setLonMax(lonMax: Double?) {
        this.lonMax = lonMax
    }

    fun getLonMin(): Double? {
        return lonMin
    }

    fun setLonMin(lonMin: Double?) {
        this.lonMin = lonMin
    }
}